.devcontainer/develop_on_codespaces.rst

```diff
@@ -0,0 +1,94 @@
+.. _develop_on_codespaces:
+
+
+Develop on Codespaces
+=====================
+
+`Codespaces <codespaces_>`_ is a virtual development environment provided by GitHub.
+You can launch a container which all the required tools and start to work
+in couple of minutes.
+This is an easy way to get started with PyMAPDL development without going
+through the process of setting your environment.
+
+.. warning:: `Codespaces <codespaces_>`_  are not free, but they have a generous
+   free allowance per month.
+   After that, you must pay according to `GitHub pricing <github_pricing_>`_.
+   You can check your billing details in your `GitHub account settings` under
+   `Billing and plans` and `Plans and usage`.
```

Comment:
UI elements should be tagged in bold rather than the single backticks. I didn't make suggest these changes because I wasn't absolutely certain that they were UI elements.

Reply:
Ohh....